ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -475,6 +490,19 @@ func (s *Service) GetTags() ([]*ecs.Tag, error) {
 	return s.tags, nil
 }
 
+func (s *Service) getLoadBalancerList(loadBalancerValue []string) ([]*ecs.LoadBalancer, error) {
+	if s.loadBalancers == nil {
+		loadBalancers := make([]*ecs.LoadBalancer, 0)
+		var err error
+		loadBalancers, err = utils.ParseLoadBalancers(loadBalancerValue, loadBalancers)
```

Comment:
you don't actually need to init an empty slice. Instead it would be cleaner to

```
loadBalancers, err := utils.ParseLoadBalancers(loadBalancerValue)
```